src/prefect/agent/ecs/agent.py

```diff
@@ -486,6 +486,14 @@ def get_run_task_kwargs(
         # - Values set using the `--env` CLI flag on the agent
         env = self.env_vars.copy()
         if run_config.env:
+            env.update(
+                {
+                    "PREFECT__LOGGING__LEVEL": run_config.env.pop(
+                        "PREFECT__LOGGING__LEVEL",
+                        env.get("PREFECT__LOGGING__LEVEL", config.logging.level),
```

Comment:
Same comment as above -- setting the env var populates the config, so defaulting to `config.logging.level` should be sufficient

Reply:
Same comment as above https://github.com/PrefectHQ/prefect/pull/4383/files/caacdb3e8693d1d1bf4e95663220e99d240a1ff2#r613232254

Reply:
Now I'm wondering why the implementation of this is different per agent